src/App.jsx

```diff
@@ -1,107 +1,19 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import { PostList } from './components/PostList';
+import { getUserById, getComments } from './utils/utils';
```

Comment:
👍

Reply:
But it would be better to change the file name to a more appropriate one instead of utils